File: Client-App/app/_model/team.ts
import { Comp } from "./comp"

export interface Team{
  id: number
  compID: number
  name: string
  pic: string
  rating: number
  compsCount: number
  club: boolean
  inComp: boolean
  hasFixture: boolean
  comps: Comp[]
  timestamped: Date
}

File: Model/Entities/Team.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Pickfc.Model.Entities
{
    public class Team
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Pic { get; set; } = string.Empty;
        public int Rating { get; set; }
        public int CompsCount { get; set; }
        public bool Club { get; set; }
        public DateTime Timestamped { get; set; }
    }
} 
